templates/zerver/help/browse-join-streams.md

```diff
@@ -0,0 +1,24 @@
+# Browse and join streams
+On Zulip, users communicate with each other by sending messages to conversation streams, which are similar to conversation threads. Users are automatically subscribed to designated streams in the realm; however, users may want to join additional streams or browse the list of streams that they've joined.
```

Comment:
May be we should use 'default streams' instead of 'designated streams'? Not sure though.